vendor.mod

```diff
@@ -0,0 +1,81 @@
+module github.com/docker/swarmkit
+
+go 1.17
+
+require (
+	code.cloudfoundry.org/clock v1.0.0
```

Comment:
did this one work with `v1.0.0` ? I recalled they tagged `1.0.0` (without `v` prefix, and therefore `go modules` didn't see that as a release); https://github.com/cloudfoundry/clock/releases/tag/1.0.0

Reply:
They've made two tags actually `1.0.0` and `v1.0.0`: https://github.com/cloudfoundry/clock/tags

Reply:
oh 🤦 how did I miss that? Perhaps I'm confused with another repository